data/colors_v2/themes/dark_high_contrast.ts

```diff
@@ -4,59 +4,118 @@ import dark from './dark'
 const scale = {
   black: '#010409',
   white: '#ffffff',
-  gray: ['#ffffff', '#f0f3f6', '#d9dee3', '#bdc4cc', '#9ea7b3', '#7a828e', '#525964', '#272b33', '#272b33', '#0a0c10'],
-  blue: ['#cce9ff', '#aedcff', '#8eccff', '#6cb8ff', '#589df3', '#1f56aa', '#0d4091', '#0c306d', '#072357', '#041844'],
-  green: ['#bcf3bc', '#88ec90', '#63de70', '#4bc95e', '#35b14d', '#1d672e', '#134f23', '#083c17', '#022d0f', '#03200c'],
+  gray: [
+    '#ffffff', 
+    '#f0f3f6', 
+    '#d9dee3', 
+    '#bdc4cc', 
+    '#9ea7b3', 
+    '#7a828e', 
+    '#525964', 
+    '#272b33', 
+    '#272b33', 
+    '#0a0c10'
+  ],
+  blue: [
+    '#caeaff', 
+    '#addcff', 
+    '#91cbff', 
+    '#71b7ff', 
+    '#409eff', 
+    '#409eff', 
+    '#318bf8', 
+    '#2672f3', 
+    '#1e60d5', 
+    '#194fb1'
+  ],
+  green: [
+    '#adf7b2',
+    '#72f07e',
+    '#4ae15e',
+    '#26cd41',
+    '#00b427',
+    '#00b427',
+    '#01a227',
+    '#068c25',
+    '#0c7725',
+    '#0d6221'
+  ],
   yellow: [
-    '#f8e4aa',
-    '#f9d265',
-    '#ebbf50',
-    '#daaa3d',
-    '#c8922b',
-    '#814d00',
-    '#613700',
-    '#4e2b00',
-    '#3c1f00',
-    '#2c1600'
+    '#fbe59e',
+    '#fbd669',
+    '#f7c843',
+    '#f0b72f',
+    '#e09b13',
+    '#e09b13',
+    '#c88508',
+    '#ae7104',
+    '#945d02',
+    '#7b4900'
   ],
   orange: [
-    '#ffe0bc',
-    '#ffcd9a',
-    '#ffb675',
-    '#f89c57',
-    '#e78037',
-    '#934010',
-    '#772d08',
-    '#5a2208',
-    '#481600',
-    '#350f00'
+    '#ffe1b4',
+    '#ffcf86',
+    '#ffb757',
+    '#fe9a2d',
+    '#e7811d',
+    '#e7811d',
+    '#d57014',
+    '#bf5e0a',
+    '#a74c00',
+    '#8f3c00'
+  ],
+  red: [
+    '#ffdedb',
+    '#ffc9c7',
+    '#ffb1af',
+    '#ff9492',
+    '#ff6a69',
+    '#ff6a69',
+    '#ff4445',
+    '#e82a2f',
+    '#cc1421',
+    '#ad0116'
   ],
-  red: ['#ffded9', '#ffcac4', '#ffb2ab', '#ff958e', '#ff6c66', '#aa2324', '#851819', '#690c0f', '#500a0d', '#400105'],
   purple: [
-    '#efdfff',
-    '#e5ccff',
-    '#d9b7ff',
-    '#ca9fff',
-    '#b384f9',
-    '#673fbb',
-    '#512d94',
-    '#3e2171',
-    '#30145d',
-    '#240e44'
+    '#f0dfff',
+    '#e6ccff',
+    '#dbb7ff',
+    '#cb9eff',
+    '#b780ff',
+    '#b87fff',
+    '#a66bff',
+    '#954ffd',
+    '#8031f7',
+    '#6921d7'
   ],
-  pink: ['#ffdbeb', '#ffc6e1', '#ffadd5', '#ee97c5', '#dd7cae', '#933869', '#752652', '#5c183f', '#4b0b31', '#390524']
+  pink: [
+    '#ffdceb', 
+    '#ffc7e1', 
+    '#ffadd4', 
+    '#ff8dc7', 
+    '#ef6eb1', 
+    '#ef6eb1', 
+    '#e456a3', 
+    '#d23d91', 
+    '#b72c7d', 
+    '#9c1d6a']
 }
 
 const exceptions = {
   fg: {
-    muted: get('scale.gray.1')
+    muted: get('scale.gray.1'),
+    onEmphasis:get('scale.gray.9')
   },
   border: {
     default: get('scale.gray.5'),
     muted: get('scale.gray.5')
   },
   neutral: {
     emphasis: get('scale.gray.6')
+  },
+  btn.primary: {
```

Comment:
Not sure if this is the correct way to call btn primary…?

Reply:
This should fix it: https://github.com/primer/primitives/pull/137#discussion_r650254548